src/Compilers/CSharp/Portable/Symbols/Source/SourceComplexParameterSymbol.cs

```diff
@@ -943,7 +943,18 @@ internal override bool IsMetadataOptional
             }
         }
 
-        internal sealed override bool IsMetadataIn => GetDecodedWellKnownAttributeData()?.HasInAttribute == true;
+        internal sealed override bool IsMetadataIn
+        {
+            get
+            {
+                if (RefKind == RefKind.In && ContainingSymbol.IsExtern)
```

Comment:
How does `ContainingSymbol.IsExtern` impact this?

Reply:
This attribute is only going to be emitted when applied to `extern` methods.